src/app/services/services.module.ts

```diff
@@ -108,8 +108,9 @@ const folderService = new FolderService(cryptoService, userService, apiService,
 const collectionService = new CollectionService(cryptoService, userService, storageService, i18nService);
 searchService = new SearchService(cipherService, platformUtilsService);
 const policyService = new PolicyService(userService, storageService);
-const lockService = new LockService(cipherService, folderService, collectionService,
-    cryptoService, platformUtilsService, storageService, messagingService, searchService, userService, null);
+const vaultTimeoutService = new VaultTimeoutService(cipherService, folderService, collectionService,
+    cryptoService, platformUtilsService, storageService, messagingService, searchService, userService, null,
+    async () => messagingService.send('logout', { expired: true }));
```

Comment:
That makes sense. While implementing, I convinced myself that arguments could be made for both sides of the conversation. Since the vault timeout could be interpreted as a user's access being 'expired'.  Regardless, will update to use false.